src/Microsoft.DotNet.Cli.Utils/project.json

```diff
@@ -1,14 +1,15 @@
 ﻿{
-    "version": "1.0.0-*",
+  "version": "1.0.0-*",
 
-    "shared": "**/*.cs",
+  "shared": "**/*.cs",
 
-    "dependencies": {
-      "NETStandard.Library": "1.0.0-rc2-23616",
-      "Microsoft.DotNet.ProjectModel": "1.0.0"
-    },
+  "dependencies": {
+    "NETStandard.Library": "1.0.0-rc2-23616",
+    "Microsoft.DotNet.ProjectModel": "1.0.0",
+    "System.Reflection.Metadata": "1.1.0"
```

Comment:
I realize this is likely not a direct change you made here, but why aren't we using the prerelease version of this? It will cause conflicts in its dependency closure. NETStandard.Library and it likely depend on different versions of `System.Reflection.Primitives`